packages/apps/package.json

```
@@ -31,10 +27,11 @@
     "eslint-config-oclif-typescript": "^0.1.0",
     "globby": "^10",
     "mocha": "^5",
-    "nyc": "^13",
+    "nock": "^13.0.1",
+    "nyc": "^15.1.0",
     "oclif": "3.4.3",
     "ts-node": "^10",
-    "typescript": "3.7.5"
+    "typescript": "4.8.4"
   },
   "engines": {
     "node": ">=10.0.0"
```

Are we still waiting for final integration to change this to 12?

---

Nope, I missed it. Thank you.
```suggestion
    "node": ">=12.0.0"
```